tracer/test/test-applications/integrations/Samples.AWS.Kinesis/SyncHelpers.cs

```diff
@@ -0,0 +1,75 @@
+#if NETFRAMEWORK
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Amazon.Kinesis;
+using Amazon.Kinesis.Model;
+
+namespace Samples.AWS.Kinesis
+{
+    static class SyncHelpers
+    {
+
+        private const string StreamName = "MyStreamName";
+
+        public static void StartKinesisTasks(AmazonKinesisClient kinesisClient)
+        {
+            Console.WriteLine("Beginning Async methods");
+            using (var scope = SampleHelpers.CreateScope("async-methods"))
```

Comment:
```suggestion
            Console.WriteLine("Beginning Synchronous methods");
            using (var scope = SampleHelpers.CreateScope("sync-methods"))
```

Reply:
Woops

Reply:
I fixed this up in my commit a87def01baf7fd2c7dfd02c43213b9203138ce0d